packages/utxo-lib/tests/__fixtures__/compose.ts

```diff
@@ -344,6 +344,77 @@ export default [
             type: 'error',
         },
     },
+    {
```

Comment:
please also add at least one valid `feeRate with decimal value` test case

Reply:
Added but not sure about the data in `transaction` 463ab037d9c46a43792a5ad5b0e6f993c4cf7d1d